exonum-java-binding/testkit/src/main/java/com/exonum/binding/testkit/TestKit.java

```diff
@@ -228,7 +354,10 @@ public Builder withTimeService(TimeProvider timeProvider) {
      */
     public TestKit build() {
       checkCorrectServiceNumber(services.size());
-      return new TestKit(services, nodeType, validatorCount, timeProvider);
+      if (nodeType == EmulatedNodeType.VALIDATOR) {
+        validatorCount += 1;
```

Comment:
I still don't think it is OK as this comment does not explain why this complication is needed.  Why don't we just set the total number of validators, not additional, and assume "1" by default?

Reply:
Ok, I'll change that. I thought it was easier for users to deal with `additional` validators and distinguish these `additional` ones as not really emulated. We would have to deal with a little more complicated internal code, but maybe it would make things a bit more straightforward for users.

Reply:
I see, but the current code does not say that anywhere but in the docs, therefore, gets confusing. Are you saying that people _need_ additional validators quite rarely as it changes almost nothing in testkit behaviour? If this distinction is important, we can leave it as is, but make it clear in names of identifiers.